<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUp</class>
 <widget class="QDialog" name="SignUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white</string>
  </property>
  <widget class="QLabel" name="SignUp_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <fontweight>Light</fontweight>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
font: 300 22pt &quot;Calibri&quot;;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt;&quot;&gt;Sign Up&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="Username_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UsernameLineEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(214, 214, 214);
	border-radius: 15px;
	color: black; 
    font-size: 16px;
    padding-left: 6px; 
}

QLineEdit::placeholder {
	color: rgb(141,141,141); 
    font-style: italic; 
}</string>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="welcome_text">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>120</y>
     <width>311</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Calibri&quot;;
background-color:transparent;</string>
   </property>
   <property name="text">
    <string>Welcome to LibManager!</string>
   </property>
  </widget>
  <widget class="QLabel" name="SignUpScreen_1">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>-30</y>
     <width>451</width>
     <height>531</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.23, y1:0.477273, x2:0.915, y2:0.0800455, stop:0 rgba(165, 34, 36, 255), stop:1 rgba(255, 224, 174, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>abalblalbablabalbalalbalblalballabalba</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>blablablablablabaalblaal</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;</string>
   </property>
   <property name="text">
    <string>blablallba</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_Button">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::StrongFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:15px;
	color: black;
	
	font: 300 10pt &quot;Calibri&quot;;
	background-color: qlineargradient(spread:pad, x1:0.23, y1:0.477273, x2:0.915, y2:0.0800455, stop:0 rgba(165, 34, 36, 255), stop:1 rgba(255, 224, 174, 255));
}</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="signupButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	border-radius:15px;
	color: black;
	
	font: 300 10pt &quot;Calibri&quot;;
	background-color: qlineargradient(spread:pad, x1:0.23, y1:0.477273, x2:0.915, y2:0.0800455, stop:0 rgba(165, 34, 36, 255), stop:1 rgba(255, 224, 174, 255));
}</string>
   </property>
   <property name="text">
    <string>Sign up</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_Line_Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(214, 214, 214);
	border-radius: 15px;
	color: black; 
    font-size: 16px;
    padding-left: 6px; 
}

QLineEdit::placeholder {
	color: rgb(141,141,141); 
    font-style: italic; 
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Surname_text">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Surname:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Surname_Line_Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(214, 214, 214);
	border-radius: 15px;
	color: black; 
    font-size: 16px;
    padding-left: 6px; 
}

QLineEdit::placeholder {
	color: rgb(141,141,141); 
    font-style: italic; 
}</string>
   </property>
   <property name="text">
    <string>Surname</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_Line_Edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(214, 214, 214);
	border-radius: 15px;
	color: black; 
    font-size: 13px;
    padding-left: 6px; 
}

QLineEdit::placeholder {
	color: rgb(141,141,141); 
    font-style: italic; 
}</string>
   </property>
   <property name="text">
    <string>Please enter a password of at least 8 characters.</string>
   </property>
  </widget>
  <widget class="QLabel" name="Mail_text">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Mail:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Mail_Line_Edit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>341</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>-1</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	background-color: rgb(214, 214, 214);
	border-radius: 15px;
	color: black; 
    font-size: 16px;
    padding-left: 6px; 
}

QLineEdit::placeholder {
	color: rgb(141,141,141); 
    font-style: italic; 
}</string>
   </property>
   <property name="text">
    <string>Mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="SignUpScreen_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>411</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(255,255,255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="roleComboBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>330</y>
     <width>82</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>User</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
  </widget>
  <zorder>SignUpScreen_2</zorder>
  <zorder>SignUpScreen_1</zorder>
  <zorder>SignUp_text</zorder>
  <zorder>Username_text</zorder>
  <zorder>UsernameLineEdit</zorder>
  <zorder>welcome_text</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>Back_Button</zorder>
  <zorder>signupButton</zorder>
  <zorder>Name_text</zorder>
  <zorder>Name_Line_Edit</zorder>
  <zorder>Surname_text</zorder>
  <zorder>Surname_Line_Edit</zorder>
  <zorder>Password_text</zorder>
  <zorder>Password_Line_Edit</zorder>
  <zorder>Mail_text</zorder>
  <zorder>Mail_Line_Edit</zorder>
  <zorder>roleComboBox</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
